docs/Gemfile.lock

```diff
@@ -30,18 +30,19 @@ GEM
       redcarpet (~> 3.1)
       safe_yaml (~> 1.0)
       toml (~> 0.1.0)
-    jekyll-coffeescript (1.0.1)
+    jekyll-coffeescript (1.0.2)
       coffee-script (~> 2.2)
+      coffee-script-source (~> 1.11.1)
     jekyll-gist (1.4.0)
-      octokit (~> 4.3.0)
+      octokit (~> 4.2)
```

Comment:
[https://github.com/jekyll/jekyll-gist/blob/master/jekyll-gist.gemspec](https://github.com/jekyll/jekyll-gist/blob/master/jekyll-gist.gemspec) In that file octokit version is 4.2.   I am sorry about I don't have much knowledge about ruby, I don't know why it's 4.2 in previous.